docs/source/developer/interactives.rst

```diff
@@ -49,29 +49,42 @@ Interactive Configuration Files
 
 - **Required Fields:**
 
-  - ``interactives:`` A dictionary of all interactive data, where each key is a slug for an interacive.
+  - ``interactives:`` A dictionary of all interactive data, where each key is a slug for an interacive. 
+TODO: Check below wording
+  - The dictionary of interactive data must include:
+
+    - ``languages:`` A dictionary of all languages the interactive is available in, where each key is the locale.
+
+    - ``is_interactive:`` A boolean value indicating if the interactive is interactable. This field was introduced
+      so that we could translate images that contain text. Therefore, if the ``is_interactive`` field is set to ``false``,
+      it means it is just an image and the user cannot interact with the content. 
```

Comment:
I don't think 'it is just an image' is correct. Perhaps say `it means the user cannot interact with the content and it is not displayed in the interactives appendix` or something

Reply:
Nice wording, thanks :)